### PR TITLE
User Guide Update - Annotations button 

### PR DESCRIPTION
**Pull request by ElenaGallo**
## Description

Add Annotations button on Annotations  section

*Documentation section involved*
- [x] User Guide
- [ ] Developer Guide

## Other useful information
Read the Docs file [here](https://elena-mapstore-docs.readthedocs.io/en/8426_userguide/user-guide/annotations/)

### CODE REVIEW

File: docs/user-guide/annotations.md [hunk 1]
```diff
@@ -1,66 +1,65 @@
-# Adding Annotations 
+# Annotations 
 ********************
```
Comment (MV88) [inline docs/user-guide/annotations.md:2]: lint error here, please fix it

File: docs/user-guide/annotations.md [hunk 2]
```diff
@@ -1,66 +1,65 @@
...
 
 [Mapstore](https://mapstore.geosolutionsgroup.com/mapstore/#/) lets you enrich the map with special features which expose additional information, mark particular position on the map and so on.
 Those features make up the so called **Annotations** layers.
 
-Starting from a new map or an already existing one:
+Starting from a new map or an already existing one, the editor can access the **Annotations** <img src="../img/button/annotations2.jpg" class="ms-docbutton" style="max-height:30px;"/> button from the [TOC](toc.md) <img src="../img/button/show-layers.jpg" class="ms-docbutton" style="max-height:30px;"/> panel on the top-left corner of the map viewer. The annotation panel will open:
```
Comment (MV88) [inline docs/user-guide/annotations.md:7]: please add here an image of the toolbar
like
image

File: docs/user-guide/annotations.md [hunk 3]
```diff
@@ -1,66 +1,65 @@
...
 
-* **Click** on the *Burger menu* button <img src="../img/button/burger.jpg" class="ms-docbutton" /> from the main menu bar.
-* **Click** on the *Annotations* option from the list <img src="../img/button/annotation-option.jpg" class="ms-docbutton" style="max-height:20px;"/>.
+<img src="../img/annotations/annotation_tool.jpg" class="ms-docimage" style="max-width:500px;" />
```
Comment (MV88) [inline docs/user-guide/annotations.md:9]: no need for this to be so long in height
image

File: docs/user-guide/annotations.md [hunk 4]
```diff
@@ -1,66 +1,65 @@
...
+Once the geometry has been saved through the **Save** <img src="../img/button/save_button.jpg" class="ms-docbutton" /> button, for each geometry created, the editor can perform the following operations:
 
-* **Click** on *Delete* <img src="../img/button/delete_white_button.jpg" class="ms-docbutton" /> to delete the geometry of the annotation.
+* **Zoom** to the geometry annotation on map through the <img src="../img/button/zoom-feature.jpg" class="ms-docbutton" /> button
 
-* **Click** on *Save* <img src="../img/button/save_button.jpg" class="ms-docbutton" /> to save the geometry information.
+* **Delete** the geometry annotation through the <img src="../img/button/delete_white_button.jpg" class="ms-docbutton" /> button
 
-* **Click** on *Save* <img src="../img/button/save_button.jpg" class="ms-docbutton" /> on the main form to save annotation information.
+Once all the *Geometries* have been created, the user can save the annotation through the **Save** <img src="../img/button/save_button.jpg" class="ms-docbutton" /> button and it will be visible:
```
Comment (MV88) [inline docs/user-guide/annotations.md:54]:
```suggestion
Once all the *Geometries* have been created, the user can save the annotation through the **Save** <img src="../img/button/save_button.jpg" class="ms-docbutton" /> button that will be visible in the annotation list:
```

File: docs/user-guide/annotations.md [hunk 5]
```diff
@@ -1,66 +1,65 @@
...
 
-The annotations layer will be added to the TOC and the annotation will be visible in the annotations list as in the figure below.
+* in the annotation list
 
-<img src="../img/annotations/tijuana_annotation.jpg" class="ms-docimage" />
+<img src="../img/annotations/annotation1.jpg" class="ms-docimage" />
+
+* once the annotations panel is closed, trought the <img src="../img/button/x2.jpg" class="ms-docbutton" /> button, in the [TOC](toc.md) as annotations layer
```
Comment (MV88) [inline docs/user-guide/annotations.md:60]:
```suggestion
Then, if not present, a new **Annotations**  layer will be created and added to the [TOC](toc.md) <img src="../img/button/x2.jpg" class="ms-docbutton" /> 
```

File: docs/user-guide/annotations.md [hunk 6]
```diff
@@ -1,66 +1,65 @@
...
+* **Zoom** to the geometry annotation on map through the <img src="../img/button/zoom-feature.jpg" class="ms-docbutton" /> button
 
-* **Click** on *Save* <img src="../img/button/save_button.jpg" class="ms-docbutton" /> to save the geometry information.
+* **Delete** the geometry annotation through the <img src="../img/button/delete_white_button.jpg" class="ms-docbutton" /> button
 
-* **Click** on *Save* <img src="../img/button/save_button.jpg" class="ms-docbutton" /> on the main form to save annotation information.
+Once all the *Geometries* have been created, the user can save the annotation through the **Save** <img src="../img/button/save_button.jpg" class="ms-docbutton" /> button and it will be visible:
 
-The annotations layer will be added to the TOC and the annotation will be visible in the annotations list as in the figure below.
+* in the annotation list
```
Comment (MV88) [inline docs/user-guide/annotations.md:56]:
```suggestion

```

File: docs/user-guide/annotations.md [hunk 7]
```diff
@@ -120,26 +119,28 @@ Click on <img src="../img/button/apply_button.jpg" class="ms-docbutton"/> to app
 Managing Annotations
 --------------------
 
-You can manage anytime your annotations through the *Annotation Viewer*. Click on some annotation to display it.
+Once annotations are added to the [TOC](toc.md), the user can **Manage** it by clicking to <img src="../img/button/edit_button.jpg" class="ms-docbutton" /> button from the TOC toolbar and the *Main Annotations panel* will be open.
```
Comment (MV88) [inline docs/user-guide/annotations.md:122]:
```suggestion
Once annotations are added to the [TOC](toc.md), the user can **Manage** them by clicking to <img src="../img/button/edit_button.jpg" class="ms-docbutton" /> button from the TOC toolbar and the *Main Annotations panel* will be open.
```

Comment (MV88) [inline docs/user-guide/annotations.md:122]: maybe we can create a new section "Main Annotations panel" similar to Managing Annotations that can be linked here?

File: docs/user-guide/annotations.md [hunk 8]
```diff
@@ -120,26 +119,28 @@ Click on <img src="../img/button/apply_button.jpg" class="ms-docbutton"/> to app
...
 
-<img src="../img/annotations/annotation_toolbar.jpg" class="ms-docimage" style="max-width:500px;"/>
+<img src="../img/annotations/annotations_main_panel.jpg" class="ms-docimage" style="max-width:500px;" />
 
-* You can edit your annotations by clicking the geometry card from list of geometries of the annotation. It allows you to change the geometry and/or the information related to the annotations. See the gif below to better understand how:
+From it, the editor is allowed to:
```
Comment (MV88) [inline docs/user-guide/annotations.md:126]: here you use the word editor and in other places the word user, can you align these to editor or user?

File: docs/user-guide/annotations.md [hunk 9]
```diff
@@ -120,26 +119,28 @@ Click on <img src="../img/button/apply_button.jpg" class="ms-docbutton"/> to app
...
+* **Show/Hide** an annotation on the map by clicking on <img src="../img/button/eyeon.jpg" class="ms-docbutton" /> button
 
-* Annotations can be also download in `json` format and reused in other maps. Using <img src="../img/button/download_annotation_button.jpg" class="ms-docbutton" /> button you will be able to export your annotation in the `Annotation.json` file and download it.
+From the *Main Annotations Panel*, by selecting an annotation from the list, the user is returned to the *Annotation Viewer* where the annotation can be edited.
 
-<img src="../img/annotations/download_annotation.jpg" class="ms-docimage"/>
+<img src="../img/annotations/annotation_toolbar.jpg" class="ms-docimage" style="max-width:500px;"/>
 
-From the main panel of the *Annotations* tool you can also download/upload annotations to/from a `json` file.
+In particular, the editor can:
```
Comment (MV88) [inline docs/user-guide/annotations.md:140]: isn't this part a bit mixed to Styling annotations, maybe you can move the gif here to that section

**Review by MV88 — CHANGES_REQUESTED** [review]
- [ ] the main menu entry on the left list is still called adding annotations
image, can you change it to Annotations ?

- [x] check lint problems on the .md files you edited, see [here](https://github.com/geosolutions-it/MapStore2/pull/8417#pullrequestreview-1115391391)

**Comment by tdipisa** [comment]
> * [ ]  the main menu entry on the left list is still called adding annotations

@ElenaGallo I think this still need to be fixed.

**Review by tdipisa — APPROVED** [review]

**Comment by tdipisa** [comment]
@ElenaGallo please backport to 2022.02.xx. Thank you

**Pull request merged**